server/routes/messaging.ts

```diff
@@ -55,21 +54,24 @@ router.route("/api/messaging/coordinator-recruitment-text").post(
   asyncHandler(async (req: Request, res: Response) => {
     if (process.env.NODE_ENV !== "production") {
       res.status(FORBIDDEN);
-      throw new Error(
-        "'NODE_ENV' must be set to 'production' to start Make automations."
+      logger.error(
+        new Error(
+          "'NODE_ENV' must be set to 'production' to start Make automations."
+        )
       );
     }
     if (!process.env.COORDINATOR_RECRUITMENT_TEXT_WEBHOOK) {
       res.status(INTERNAL_SERVER_ERROR);
-      throw new Error("'COORDINATOR_RECRUITMENT_TEXT_WEBHOOK' not set.");
+      logger.error(
+        new Error("'COORDINATOR_RECRUITMENT_TEXT_WEBHOOK' not set.")
+      );
     }
 
-    const resp = await fetch(process.env.COORDINATOR_RECRUITMENT_TEXT_WEBHOOK);
+    const resp = await fetch(
+      `${process.env.COORDINATOR_RECRUITMENT_TEXT_WEBHOOK}`
```

Comment:
Was this a bug before? Why do we have raw `process.env.<something>` throughout and also cases of using \`${process.env.<something>}\`?

Reply:
Backticks have been removed to use raw `process.env.<>` now. I changed it because I mistakenly was removing the throws which was causing a compiler error. Using the backticks allowed me to work around the error, but as I finally understand what I've be doing wrong, the backticks are unnecessary.